maven-model-builder/src/site/apt/index.apt

```diff
@@ -184,6 +184,8 @@ Maven Model Builder
 *----+------+------+
 | <<<maven.build.version>>> | The full build version of the current Maven execution <(since 3.0.4)>. For example, "<<<Apache Maven 3.2.2 (r01de14724cdef164cd33c7c8c2fe155faf9602da; 2013-02-19T14:51:28+01:00)>>>". | <<<$\{maven.build.version\}>>> |
 *----+------+------+
+| <<<maven.repo.local>>> | The repository on the local machine Maven should use to store downloaded artifacts (jars etc). | <<<$\{${user.home}/.m2/repository\}>>> |
```

Comment:
Something is wrong here: `<<<$\{${user.home}/.m2/repository\}>>>`. Too many braces and dollars.

Reply:
I removed the redundant ones.